docs/analytics/configuration.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Configuration
+---
+
+The `duckdb_execute` function can be used to change the underlying [configuration options](https://duckdb.org/docs/configuration/overview#global-configuration-options)
+used by DuckDB.
+
+```sql
+-- Dollar quoted strings are used to escape single quotes
+SELECT duckdb_execute($$SET memory_limit='10GiB'$$);
+```
+
+`duckdb_settings` returns a table of all available settings.
+
+```sql
+SELECT * FROM duckdb_settings();
+```
+
+<Note>
+  Because a new DuckDB connection is created per Postgres connection, every new
+  Postgres connection uses the default DuckDB configuration. Changes to the
+  DuckDB configuration only apply to the current Postgres connection.
+</Note>
```

Comment:
I want to double-check that this is an accurate statement, as it seem that DuckDb has [configuration scopes](https://duckdb.org/docs/sql/statements/set.html#scopes) that can be used along with `SET`.

Your `duckdb_execute` function seems flexible enough to accommodate `SET GLOBAL`, but if I want to make sure that we're not misleading the user with this:

> Changes to the DuckDB configuration only apply to the current Postgres connection

Reply:
It is indeed, I tested it with SET GLOBAL as well. This is because our entire DuckDB connection is in memory, `SET GLOBAL` would only work if we were connecting to a `.db3` database on disk.